docs/samples.md

```diff
@@ -0,0 +1,94 @@
+# Presidio API Samples
```

Comment:
There's an index.md file with links to all documents. This allows us to expose this doc: https://microsoft.github.io/presidio/
We need to make sure that all docs are properly referenced there and (if applicable) in the README.md